src/Dropdown/Option.jsx

```diff
@@ -4,13 +4,18 @@ import styled from 'styled-components'
 import css from '@styled-system/css'
 
 import { typography } from '../Typography'
+import Flex from '@ivoryio/kogaio/Responsive/Flex'
+import Box from '@ivoryio/kogaio/Responsive/Box'
+import { Checkbox } from '../Checkbox'
+import sizes from '../assets/sizes'
 
 const DropdownItem = ({
   children,
   onSelectItem,
   disabled,
   isSelected,
-  value
+  value,
+  multiple
```

Comment:
The DropdownItem should not be aware of a `multiple` prop. You might want to rename it `hasCheckbox, which is something the DropdownItem should be aware of.